bootstrap.sh

```diff
@@ -21,34 +14,16 @@ missingCommand() {
 }
 
 # First Check to see if they have the neccessary software installed
-command -v carthage >/dev/null 2>&1 || { missingCommand "carthage" "https://github.com/Carthage/Carthage/releases"; }
 command -v swiftlint >/dev/null 2>&1 || { missingCommand "swiftlint" "https://github.com/realm/SwiftLint/releases"; }
 command -v npm >/dev/null 2>&1 || { missingCommand "npm" "https://nodejs.org/en/download/"; }
 
-IS_CI_BUILD=0
-
 # Log Colors
 COLOR_ORANGE='\033[0;33m'
 COLOR_NONE='\033[0m'
 
-for i in "$@"
-do
-case $i in
-    -f|--force)
-    rm -rf Carthage/*
-    rm -rf ~/Library/Caches/org.carthage.CarthageKit
-    shift
-    ;;
-    --ci)
-    IS_CI_BUILD=1
-    shift
-    ;;
-esac
-done
-
 # Install Node.js dependencies and build user scripts
 
-if [ "$IS_CI_BUILD" = 0 ]; then
+if [ "$1" == --ci ]; then
```

Comment:
This is probably fine for now, but we may have to bring back the switch-case & var above if we ever add another argument

Reply:
Do you want to go back to previous version and just remove the carthage part? i'm ok with both ways

Reply:
Nah its fine, we can always revisit later anyways